ellis/Dockerfile

```diff
@@ -5,6 +5,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-
 RUN /etc/init.d/mysql start && apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes ellis
 RUN /etc/init.d/mysql start
 
+COPY create_numbers.sh /usr/share/clearwater/ellis/create_numbers.sh
```

Comment:
ellis is trying to find `http://hs.default.svc.cluster.local:8889`, which I think should be `http://headstead.default.svc.cluster.local:8889`